images/build/Dockerfiles/Dockerfile

```diff
@@ -47,21 +47,24 @@ RUN set -ex \
     && ln -s $DOT_NET_CORE_22_SDK_VERSION 2.2 \
     && cd $imagesDir \ 
     && . $buildDir/__nodeVersions.sh \
-    && $imagesDir/installPlatform.sh nodejs $NODE8_VERSION \
-    && $imagesDir/installPlatform.sh nodejs 8.0.0 \
-    && $imagesDir/installPlatform.sh nodejs 8.1.4 \
-    && $imagesDir/installPlatform.sh nodejs 8.2.1 \
-    && $imagesDir/installPlatform.sh nodejs 8.8.1 \
-    && $imagesDir/installPlatform.sh nodejs 8.9.4 \
-    && $imagesDir/installPlatform.sh nodejs 8.11.4 \
-    && $imagesDir/installPlatform.sh nodejs 8.12.0 \
-    && $imagesDir/installPlatform.sh nodejs 8.15.1 \
-    && $imagesDir/installPlatform.sh nodejs 9.4.0 \
-    && $imagesDir/installPlatform.sh nodejs 10.1.0 \
-    && $imagesDir/installPlatform.sh nodejs 10.10.0 \
-    && $imagesDir/installPlatform.sh nodejs 10.14.2 \
-    && $imagesDir/installPlatform.sh nodejs 10.21.0 \
-    && $imagesDir/installPlatform.sh nodejs 12.18.2 \
+    && . $buildDir/__sdkStorageConstants.sh \
+    && export DYNAMIC_INSTALL_ROOT_DIR="/opt" \
```

Comment:
just curious .. aren't we using /tmp for this location? why choosing /opt?

Reply:
This is for baked in platforms in the image which we install under `/opt`